### PR TITLE
Add comt pages and routing

### PR DESCRIPTION
**Pull request by gitchrisadams**

### CODE REVIEW

**Comment by gitchrisadams** [comment]
CC: @Bobfrat

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM debian:jessie
+MAINTAINER Luke Campbell <luke.campbell@rpsgroup.com>
```
Comment (Bobfrat) [inline Dockerfile]: Let's change this to Ben Adams <ben.adams@rpsgroup.com>

File: bin/www [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+var app = require('../app');
+var debug = require('debug')('ioos-comt:server');
+var http = require('http');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3001');
```
Comment (Bobfrat) [inline bin/www]: Let's go with 3000 here by default

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ioos-us",
+  "version": "0.0.1",
+  "homepage": "https://github.com/ioos/ioos-us",
+  "authors": [
+    "Luke Campbell <luke.s.campbell@gmail.com>"
+  ],
+  "description": "The homepage for http://ioos.us/",
+  "main": "",
```
Comment (Bobfrat) [inline bower.json]: can you update the metadata above?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ioos-us",
```
Comment (Bobfrat) [inline package.json]: comt-landing

File: bower.json (hunk 3 above)
Comment (gitchrisadams) [reply bower.json]: Lock down all versions so no ^ carrot? In npm and bower?

Comment (gitchrisadams) [reply bower.json]: Preference on package.json name. Currently name: "ioos-us"

Comment (Bobfrat) [reply bower.json]: comt-landing

File: routes/index.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    projectTitle: projectTitle,
+    title_key: req.params.title_key,
+    subProjectTitle: req.query.t
+  });
+});
+
+/* GET comt model viewer redirect. */
+router.get('/model_viewer', function(req, res, next) {
+  res.writeHead(301,
+    {Location: 'http://oceansmap.com/comt/'}
```
Comment (Bobfrat) [inline routes/index.js:112]: @gitchrisadams I think we'll want to update this to comt.ioos.us/map

Comment (gitchrisadams) [reply routes/index.js:112]: Why update it to comt.ioos.us/map? When visit comt.ioos.us/map it redirects to oceansmap.com/comt  What about updating the links on page to linking to comt.ioos.us such as the map image and the viewers dropdown > Coastl and Ocean Modeling Testbed ?

**Pull request merged**